### ISSUE TITLE
Found old "program" version in "C:\...", but not using....

### ISSUE THREAD

**Issue by pulse-wave** [cmderdev/cmder#1479]
I have installed cmder minimal is a great piece of software, though I would like to know if there is a remedy for this:

```
Found old git version in "C:\ProgramData\scoop", but not using...

D:\Users\pulse
λ
```

I've installed scoop too and git-with-openssh with it (scoop.sh).
Every time I start up cmder it complains about an old git version (that is not in the directory it says, plus is the latest), not a big deal but I'm wondering, is there any hack I can do to not let cmder shows the warning at every startup?

Not a big deal really just a bit annoying it keeps repeating it every single time (it's also wrong I guess because git is not in that directory and is the latest)

Thank you in advance

**Comment by anaschillin** [comment]
Try this: https://gist.github.com/n3dst4/cc875d83401c06bd6faa

**Comment by pulse-wave** [comment]
I'm sorry @anaschillin probably I'm not understanding you very well, but I looked at the link you posted...the problem is I don't have installed git bash I don't use it, I use git-for-windows, plus I would like just to avoid that warning message everytime I open cmder is not related to when I use git only, it appears everytime I start cmder.

Your suggestion seems to create a task just for git bash?! 

Am I missing something here?

**Comment by anaschillin** [comment]
Oh  I see, I misunderstood. You can try unistalling scoop and git-for-windows and install them through chocolatey ...

**Comment by pulse-wave** [comment]
I will try thanks

**Comment by srend42** [comment]
One workaround (while keeping scoop) is to create and set a `GIT_INSTALL_ROOT` environment variable that points to the current Git Scoop directory, i.e.: `C:\Users\USERNAME\scoop\apps\git-with-openssh\current` (change username to your Windows user name)

**Comment by DRSDavidSoft** [comment]
I believe a fix for #1564 in general would solve this issue and related unix flavored git problems.
